src/explore-education-statistics-frontend/src/modules/download/DownloadIndexPage.tsx

```diff
@@ -57,7 +58,15 @@ const DownloadIndexPage: NextPage<Props> = ({ themes = [] }) => {
       </div>
 
       {themes.length > 0 ? (
-        <Accordion id="downloads">
+        <Accordion
+          id="downloads"
+          onSectionOpen={accordionSection => {
+            logEvent(
+              'Download index page',
+              `${accordionSection.title} accordion opened`,
```

Comment:
I think we should try to be consistent and use the same format when tracking any accordion i.e.
- category: [the page]
- action: [accordion name if applicable] accordion section opened
- label: [section]

The same applies for the rest of the accordions in this PR.